accepted/future-releases/small-features-21Q1/feature-specification.md

```diff
@@ -0,0 +1,109 @@
+# Dart "Small Features" 21Q1
+
```

Comment:
Add a CHANGELOG at the top?  I found it very useful with the null safety feature spec.

Reply:
ACK